src/protocol/attribution/aggregate_credit.rs

```diff
@@ -55,20 +55,20 @@ where
 /// Transpose rows of bits into bits of rows
 ///
 /// input:
-/// [
-///   [ row[0].bit0, row[0].bit1, ..., row[0].bit31 ],
-///   [ row[1].bit0, row[1].bit1, ..., row[1].bit31 ],
+/// `\[`
+///  `\[`row`\[`0`\]`.bit0,row`\[`0`\]`.bit1, ...,row`\[`0`\]`.bit31`\]`
```

Comment:
I *think* you can just put one backtick at the start and one at the end, and treat the entire line as a math expression.

Reply:
Thanks for this! I wasted some time to get the backticks in :(